### PR TITLE
Fixed Various Issues including Exam Inventory Table and Rescheduling …

### PR DESCRIPTION
**Pull request by scottrumsby**
…Modal

Fixed logic on exam inventory table so that...
-The "Still Required" details are correct
-The Yellow and Green Icons display correctly

Fixed Issue with the Edit Exam Modal date being stuck after setting exam_received to yes

Fixed issue with exam_type select field in Edit Exam modal

Fixed an issue with the Edit Group Booking modal where the data could show as the previous day for an exam booked in the morning.

Removed more instances of 'challenger' vs. 'monthly session exam'

Changed all uses of .includes('Group') to use the group_exam_ind flag

### CODE REVIEW

File: frontend/src/store/index.js [hunk 1]
```diff
@@ -481,6 +481,9 @@ export const store = new Vuex.Store({
 
     putBooking(context, payload) {
       return new Promise((resolve, reject) => {
+        if (payload.changes && payload.changes.start_time) {
```
Comment (sjrumsby) [inline frontend/src/store/index.js]: Empty if block here?

File: frontend/src/exams/exam-inventory-table.vue [hunk 2]
```diff
@@ -177,33 +177,11 @@
...
-                     class="ml-3 mt-1">Event ID</div>
-              </template>
-              <template v-else>
-                <div v-if="!row.item.booking"
-                     class="ml-3 mt-1">Scheduling</div>
-                <div
-                  v-if="row.item.booking && !checkInvigilator(row.item)"
-                  class="ml-3 mt-1">Assignment of Invigilator</div>
+              <template v-for="req in stillRequires(row.item)">
+                <div key="i+'it'" class="ml-3 mt-1">{{ req }}</div>
```
Comment (sjrumsby) [inline frontend/src/exams/exam-inventory-table.vue]: Well this is just a little cleaner

File: frontend/src/exams/edit-exam-form-modal.vue [hunk 3]
```diff
@@ -301,14 +298,12 @@
         return ''
       },
       examType() {
-        if (this.examRow && this.examRow.exam_type) {
```
Comment (sjrumsby) [inline frontend/src/exams/edit-exam-form-modal.vue]: Was `this.examRow` removed in a previous PR or something? I don't see it anywhere in this file.

Comment (scottrumsby) [reply frontend/src/exams/edit-exam-form-modal.vue]: examRow was turned into exam a long time ago so this function has been else'ing for a long time

**Pull request merged**